### ISSUE TITLE
Make Problem statements fully extensible

### ISSUE THREAD

**Issue by julian-christl** [ls1intum/Artemis#1943]
### Is your feature request related to a problem?
Inside the exam it's nearly impossible to extend the problem statement to the left to be able to read it more easily

### Describe the solution you'd like
A button which extends the problem statement fully to the left (and back again)

### Describe alternatives you've considered
Making it possible to open the problem statement in a new window but that's a whole other level which will surely not be realised before the exam.

### Additional context
Especially a problem with programming exercises.
Might be a thing we can support in the normal exercises as well?
Critical because I think it's quite a handicap currently (especially for smaller screens) and it shouldn't be that much work if I'm correct.

**Comment by krusche** [comment]
I don't consider this as critical. Please be careful with this priority.

Students can still read the problem statement and work on the exercises.
Changing the whole page layout might be too risky at the moment. 
I agree: for the future, we should definitely improve this.

**Comment by krusche** [comment]
By the way: I added screenshots into the tutorial how to resize the problem statement and how to make it as large as possible for programming exercises with online editor. 

<img width="1096" alt="image" src="https://user-images.githubusercontent.com/744067/87934292-1209fd00-ca8f-11ea-9b1f-617246fc9b49.png">

As you can see in the screenshot, the problem statement makes 75% of the screen width. I think this is good enough so that students can read it

**Comment by julian-christl** [comment]
Sorry for the prio :D The students reported that there weren't able to resize this as much as you did here but if it's possible then it's not a critical prio indeed.

**Comment by krusche** [comment]
I guess the easiest "fix" would be to offer another button/tabs to switch between "Online Editor Mode" and "Local IDE Mode". In the Local IDE Mode, the page would only show the problem statement and would even hide the Save and Submit buttons.

The user interface already has quite some buttons (the screenshot misses the clone button), and adding even more buttons does not make it easier.

However, students are now used to this mode and might be confused in the actual GOE, so I would like to avoid changes that are not 100% needed

**Comment by LorenzNickel** [comment]
Hey, just sliding in here because I was the one complaining in Slack about this. On your screenshot in [this reply](https://github.com/ls1intum/Artemis/issues/1943#issuecomment-660978273) the problem statement indeed is quite big compared to the size of the screen. However, I have probably a bigger monitor than in the screenshot (and it's not that big, it's 1920x1080 pixel) and I was maybe able to scale the problem statement to half of the screen.

Here's a GIF of what exactly the problem for me was (hit me up in Slack for better quality and the uncensored version if needed):
![artemis-resize](https://user-images.githubusercontent.com/29959150/87985210-8074ac80-cadb-11ea-815a-07782a07a253.gif)
You can also see the scrollbar of the problem statement on the right which indicates that a lot of the problem statement has to be read by scrolling down which is annoying especially since I have so much space which I do not use on the left anyway (as I'm cloning the repo and not using the online editor).


So to summarize, it would already be a big help, if the right window could be extended a bit further to the left if that's not too much effort.

PS: Chrome 83.0.4103.116 - Windows 10

**Comment by krusche** [comment]
@LorenzNickel If you have such a wide monitor, the problem statements should be large enough to read ;-)
Scrolling vertically ist mostly needed because the content is formatted with line breaks.

Anyway, I was able to find the place where the max width is specified. The max width was set to half the browser window width which is in line with your finding.

I now changed it to ~75% of the screen, see 9c084f8e7891243678360cd6feb2588f98f252b3
This should already improve the problem that you are facing.

Larger values are problematic because then the window expands to the right over the border of the main Artemis window.

We will deploy the next release 4.3.6 tomorrow (Wednesday) morning. Please try it out afterwards and let me know if this works for you.

**Comment by krusche** [comment]
I close this for now, because the main reason behind the issue is fixed.
